src/lib/sourcepointConfig.ts

```diff
@@ -0,0 +1,3 @@
+export const ACCOUNT_ID = 1257;
+export const VENDOR_LIST_ID_CCPA = '5eba7ef78c167c47ca8b433d';
+export const CMP_ID_TCFV2 = 106842;
```

Comment:
These variable names could be more descriptive.
```suggestion
export const CCPA_PRIVACY_MANAGER_ID = '5eba7ef78c167c47ca8b433d';
export const TCFV2_VENDORLIST_ID = 106842;
```

Reply:
Agreed, updated now.